tests/tests_pytorch/strategies/test_ddp_strategy.py

```diff
@@ -276,3 +276,23 @@ def test_ddp_strategy_checkpoint_zero_redundancy_optimizer(tmpdir, strategy):
     # Assert model parameters are identical after loading
     for trained_param, loaded_param in zip(model.parameters(), saved_model.parameters()):
         assert torch.equal(trained_param.to("cpu"), loaded_param)
+
+
+class UnusedParametersModel(BoringModel):
+    def __init__(self):
+        super().__init__()
+        self.intermediate_layer = torch.nn.Linear(32, 32)
+
+    def training_step(self, batch, batch_idx):
+        with torch.no_grad():
+            batch = self.intermediate_layer(batch)
+        return super().training_step(batch, batch_idx)
+
+
+def test_ddp_strategy_find_unused_parameters_exception():
+    """Test that the DDP strategy can change PyTorch's error message so that it's more useful for Lightning
+    users."""
+    # TODO: Update default
+    trainer = Trainer(accelerator="cpu", devices=1, strategy="ddp_find_unused_parameters_false", max_steps=2)
```

Comment:
TODO after https://github.com/Lightning-AI/lightning/pull/16611 gets merged
```suggestion
    # TODO: Update default
    trainer = Trainer(accelerator="cpu", devices=1, strategy="ddp", max_steps=2)
```